src/Sushi.php

```diff
@@ -88,8 +88,24 @@ public function migrate()
             }
 
             foreach ($firstRow as $column => $value) {
-                $type = is_numeric($value) ? 'integer' : 'string';
-
+                 
+                switch (true) {
+                    case is_numeric($value):
+                        $type = 'integer';
```

Comment:
`is_numeric()` returns also `true` for floats.
https://www.php.net/manual/en/function.is-numeric.php

```suggestion
                    case is_int($value):
                        $type = 'integer';
                        break;
                    case is_numeric($value):
                        $type = 'float';
```